rust/crd/src/druidconnection.rs

```diff
@@ -0,0 +1,88 @@
+use serde::{Deserialize, Serialize};
+use stackable_operator::k8s_openapi::apimachinery::pkg::apis::meta::v1::Time;
+use stackable_operator::k8s_openapi::chrono::Utc;
+use stackable_operator::kube::CustomResource;
+use stackable_operator::kube::ResourceExt;
+use stackable_operator::schemars::{self, JsonSchema};
+
+#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ClusterRef {
+    pub name: String,
+    pub namespace: String,
```

Comment:
Im sorry i suggested it is this way. The way to go should be an optional namespace.
(I stumbled over that while adapting KUTTL tests, which run in random namespaces). We should "default" to the superset cluster namespace if the `ClusterRef` namespace is `None`.
You can have a look here: https://github.com/stackabletech/trino-operator/pull/95
I had to adapt a lot in Trino to make it namespace independent.